codecov.yaml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "shuup/*/management/commands/*"  # ignore management commands
```

Comment:
This is cheating. I don't like it. They are either tested or they're not. Making them disappear from the coverage reports just causes confusion.